library/NotificationCenter/Model/Message.php

```diff
@@ -62,6 +62,17 @@ public function send(array $arrTokens, $strLanguage = '', array $arrAttachments
             }
         }
 
+        $cpLanguage = $cpLanguage === '' ? $GLOBALS['TL_LANGUAGE'] : $cpLanguage;
```

Comment:
```suggestion
        $cpLanguage = $cpLanguage ?: $GLOBALS['TL_LANGUAGE'];
```

Reply:
Do we need this line at all? I thought what we want to achieve is to NOT use the request locale but the locale defined in the language model.

Reply:
Yes, but it can be the the request language is not available and then the fallback will be used!